Source/ACE/Command/Handlers/AdminCommands.cs

```diff
@@ -87,15 +89,117 @@ public static void HandleBanlist(Session session, params string[] parameters)
+                if (parameters[0].ToLower()[0] == bootType.ToLower()[0])
+                {
+                    // If found, selectorType will hold the correct AccoutLookupType
+                    if (Enum.TryParse(bootType, out selectorType))
+                    {
+                        // reaching this scope means we were successful, so we can stop looping:
+                        break;
+                    }
+                }
+            }
 
-            // TODO: output
+            // Peform logic
+            if (selectorType != AccountLookupType.Undef)
+            {
+                // Extract the name from the parameters
+                if (selectorType == AccountLookupType.Account || selectorType == AccountLookupType.Character)
+                {
+                    // Get the first name
+                    bootName = Common.Extensions.CharacterNameExtensions.StringArrayToCharacterName(parameters, 1);
+                }
+
+                switch (selectorType)
+                {
+                    case AccountLookupType.Account:
+                        {
+                            playerSession = WorldManager.Find(bootName);
+                            if (playerSession != null)
+                                bootId = playerSession.Player.Guid.Low;
+                            break;
+                        }
+                    case AccountLookupType.Character:
+                        {
+                            playerSession = WorldManager.FindByPlayerName(bootName);
+                            if (playerSession != null)
+                                bootId = playerSession.Player.Guid.Low;
+                            break;
+                        }
+                    case AccountLookupType.Iid:
+                        {
+                            // Extract the Id from the parameters
+                            uint.TryParse(parameters[1], out bootId);
+                            playerSession = WorldManager.Find(new ObjectGuid(bootId));
+                            if (playerSession != null)
+                                bootName = playerSession.Player.Name;
+                            break;
+                        }
+                }
+
+                if (playerSession != null)
+                {
+                    string bootText = $"account or player: {bootName} id: {bootId}";
+                    // Boot the player
+                    playerSession.BootPlayer();
+
+                    // Send an update to the admin, but prevent sending too if the admin was the player being booted
+                    if (session != null)
+                    {
+                        // Log the player who initiated the boot
+                        bootText = "Player: " + session.Player.Name + " has booted " + bootText;
+                        if (session != playerSession)
+                            session.Network.EnqueueSend(new GameMessageSystemChat(bootText, ChatMessageType.Broadcast));
```

Comment:
I agree and I would like to move change to a "//TODO:" for a later time. We should wait until after the server group chat foundation has been built out, too support this type of messaging.